gateway/src/endpoints/inference.rs

```diff
@@ -92,7 +120,7 @@ pub async fn inference_handler(
         http_client,
         clickhouse_connection_info,
     }): AppState,
-    StructuredJson(params): StructuredJson<Params>,
+    StructuredJson(params): StructuredJson<Params<'static>>,
```

Comment:
Skill issue on my side: how is this `'static`?

Reply:
it's static because they're always owned here -> each of the API keys is a Cow <'a, SecretString> but it is only a reference if you grab it from env at runtime and it is static.